srml/support/src/dispatch.rs

```diff
@@ -1126,6 +1127,22 @@ macro_rules! decl_module {
 					$call_type::__PhantomItem(_, _) => { unreachable!("__PhantomItem should never be used.") },
 				}
 			}
+			fn priority(&self, _len: usize) -> $crate::dispatch::TransactionPriority {
+				match self {
+					$( $call_type::$fn_name(..) => $crate::dispatch::Weighable::priority(&$weight, _len), )*
+					$call_type::__PhantomItem(_, _) => { unreachable!("__PhantomItem should never be used.") },
```

Comment:
This should be ensured at compile-time.  For example, `__PhantomItem` could contain `Infallible`, and then the `unreachable!` could be replaced by a pattern-match against it.

Reply:
@bkchr this pattern can be replaced probably everywhere in storage macros? I borrowed the `unreachable!()` from other places in the code.